### PR TITLE
Add k8s secrets & Jenkinsfile

### PR DESCRIPTION
**Pull request by danielogen**
#### Summary of Changes (What does this PR do?)
- This PR adds k8s env configuration and Jenkinfile to frontend
- Add Helm, Charts and Tiller to frontend deployment 

#### Is this change ready to hit production in its current state?
- [x] Yes
- [x] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [x] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
N/A
#### What are the relevant tickets?
N/A

### CODE REVIEW

File: k8s/prod-airqo-platform-frontend.yaml [hunk 1]
```diff
@@ -23,6 +23,37 @@ spec:
           ports:
```
Comment (Baalmart) [inline k8s/prod-airqo-platform-frontend.yaml:23]: I keep on wondering, is there a way we can explore the usage of one config file for each sub-repo and just change the specific ENVS for the different environments? More in the angle of using Helm charts? More: https://codefresh.io/helm-tutorial/helm-deployment-environments/

Comment (Baalmart) [reply k8s/prod-airqo-platform-frontend.yaml:23]: Same case for the Jenkins files......

File: jenkins/platform-prod/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,75 @@
+pipeline {
+    agent any
+    environment {
```
Comment (Baalmart) [inline jenkins/platform-prod/Jenkinsfile:3]: Just curious about why one Jenkinsfile is inside the Netmanager folder and others are at the base of the entire monorepo....no opinion/suggestion yet, just curious.

**Review by Baalmart — COMMENTED** [review]
All looks great, amazing work!

I have  just left a few comments for your review whenever you get time. Thanks @danielogen !

File: jenkins/platform-prod/Jenkinsfile (hunk 2 above)
Comment (danielogen) [reply jenkins/platform-prod/Jenkinsfile:3]: Good point @Baalmart , let me clean it up

File: k8s/prod-airqo-platform-frontend.yaml (hunk 1 above)
Comment (danielogen) [reply k8s/prod-airqo-platform-frontend.yaml]: @Baalmart, I have added helm chart capability

**Pull request merged**